cps/main.c

```diff
@@ -1067,6 +1069,16 @@ run_cps(struct net_config *net_conf, struct gk_config *gk_conf,
 		goto out;
 	}
 
+	if ((net_conf->front.hw_filter_ntuple &&
+			!is_valid_burst_config(&net_conf->front,
+			cps_conf->cps_max_pkt_burst)) ||
+			(net_conf->back.hw_filter_ntuple &&
+			!is_valid_burst_config(&net_conf->back,
+			cps_conf->cps_max_pkt_burst))) {
```

Comment:
Can't/shouldn't we keep this verification only for GT and GK block?

Reply:
I will change this part as needed once we have a conclusion on it.

Reply:
If needed, help Cody to figure out this issue because it's going to hold back this pull request.